db/migrate/20181008065816_add_kubernetes_service_link_to_service.rb

```diff
@@ -0,0 +1,5 @@
+class AddKubernetesServiceLinkToService < ActiveRecord::Migration
+  def change
+    add_column :services, :kubernetes_service_link, :string, null: true
```

Comment:
Why this requires online migration? Is this a blocking operation? I don't think so.

Reply:
It does not.

Reply:
Then https://github.com/3scale/infrastructure/issues/560 is not needed?

https://github.com/ankane/strong_migrations would detect this automatically for us.

Reply:
@mikz Then let's make that an automatic check for real because right now we look at it manually and last week we messed up and realised a migration was blocking when we had almost no time left and operations had to do our migration urgently without any previous planning because we didn't tell them with time because we first mistakenly thought it was not blocking.
Right now if we create the issue and it is not blocking, the can simply close it and no bigger damage is done. Although IMO it should be enough with an Slack message to @maneta , but as they prefer...

Reply:
This departure looks like something we should take a look as it uses the very same tool we use in ops